design/media_types.go

```diff
@@ -129,15 +129,18 @@ var workItemType = a.MediaType("application/vnd.workitemtype+json", func() {
 	a.Attribute("version", d.Integer, "Version for optimistic concurrency control")
 	a.Attribute("name", d.String, "User Readable Name of this item type")
 	a.Attribute("fields", a.HashOf(d.String, fieldDefinition), "Definitions of fields in this work item type")
+	a.Attribute("parentPath", d.String, "The names of the parent types separated with '/' or just '/' if no parents")
```

Comment:
I really dont' think this should be part of this PR. We can discuss the utility of having this exposed, but in the least, it has nothing to do with the task at hand.